### PR TITLE
doc: change references of ci.adoptopenjdk.net to ci.adoptium.net

### PR DESCRIPTION
**Pull request by gdams**
Linked to https://github.com/adoptium/infrastructure/issues/2932

##### Checklist

- [x] commit message has one of the [standard prefixes](https://github.com/adoptium/infrastructure/blob/master/CONTRIBUTING.md#commit-messages)
- [ ] [faq.md](https://github.com/adoptium/infrastructure/blob/master/FAQ.md) updated if appropriate
- [ ] other documentation is changed or added (if applicable)
- [ ] playbook changes run through [VPC](https://ci.adoptopenjdk.net/view/Tooling/job/VagrantPlaybookCheck/) or [QPC](https://ci.adoptopenjdk.net/view/Tooling/job/QEMUPlaybookCheck/) (if you have access)
- [ ] VPC/QPC not applicable for this PR
- [ ] for inventory.yml changes, bastillion/nagios/jenkins updated accordingly

### CODE REVIEW

File: FAQ.md [hunk 1]
```diff
@@ -223,7 +223,7 @@ to validate them.
 
 ## Jenkins access
 
-The AdoptOpenJDK Jenkins server at [https://ci.adoptopenjdk.net](https://ci.adoptopenjdk.net) is used for all the
+The AdoptOpenJDK Jenkins server at [https://ci.adoptium.net](https://ci.adoptium.net) is used for all the
```
Comment (sxa) [inline FAQ.md:226]: There may be more of these ...
```suggestion
The Adoptium Jenkins server at [https://ci.adoptium.net](https://ci.adoptium.net) is used for all the
```

**Comment by sxa** [comment]
This should have a VPC run to ensure there are no side effects with the changes in the playbooks. It's low risk, but let's be sure.

**Comment by gdams** [comment]
> this should have a VPC run to ensure there are no side effects with the changes in the playbooks. It's low risk, but let's be sure.

@sxa I've had a look at the VPC job but it makes no sense to me at all, for starters, it's running of @Haroon-Khel's fork and I'm not convinced I should be changing this config, the default fork is also set to `AdoptOpenJDK` rather than `Adoptium` so this will all need updating. If you'd like to kick off a VPC run for this PR then please feel free.

**Comment by sxa** [comment]
> > this should have a VPC run to ensure there are no side effects with the changes in the playbooks. It's low risk, but let's be sure.
> @sxa I've had a look at the VPC job but it makes no sense to me at all, for starters, it's running of @Haroon-Khel's fork and I'm not convinced I should be changing this config, the default fork is also set to `AdoptOpenJDK` rather than `Adoptium` so this will all need updating. If you'd like to kick off a VPC run for this PR then please feel free.

It'll handle adoptium vs adoptopenjdk without problems. I'm not sure why it's still using Haroon's fork though ...

**Comment by sxa** [comment]
https://ci.adoptium.net/view/Tooling/job/VagrantPlaybookCheck/1606/ launched with the first two parameters set to this PR branch.

**Comment by Haroon-Khel** [comment]
Rerunning the VPC for this branch https://ci.adoptium.net/job/VagrantPlaybookCheck/1608/console

I've changed the config back to use Adoptium/master. I think I was testing something and forgot to change it back when the changes got merged. 

However, the VPC runs from feb 15 and back were using my branch and they passed, so not sure if the recent failures are to do with the config using my branch

**Comment by Haroon-Khel** [comment]
VPCs are still failing, for this reason I believe
`ERROR! the field 'hosts' is required but was not set`

We were catching this error in our linter a while back, but I thought we fixed it

**Comment by Haroon-Khel** [comment]
I suspect it is the result of this pr https://github.com/adoptium/infrastructure/pull/2940/files

[- name: Ansible Unix playbook
  hosts: "{{ Groups | default('localhost:build:test:perf:jck:!*zos*:!*win*:!*aix*') }}"](https://github.com/steelhead31/infrastructure/blob/e5d7c2432f6c41c37669bd7e3e311ade74a62752/ansible/playbooks/AdoptOpenJDK_Unix_Playbook/main.yml#L8)

**Comment by steelhead31** [comment]
> I suspect it is the result of this pr https://github.com/adoptium/infrastructure/pull/2940/files

Line 239 in the vagrantplaybook.sh script needs changing as below :

From  
`sed -i -e "s/.*hosts:.*/- hosts: all/g" playbooks/AdoptOpenJDK_Unix_Playbook/main.yml`  
To  
`sed -i -e "s/.*hosts:.*/  hosts: all/g" playbooks/AdoptOpenJDK_Unix_Playbook/main.yml`

to allow for the new linter / ansible structure rules.

**Comment by Haroon-Khel** [comment]
@steelhead31 I believe you fixed the linter error that was causing `ERROR! the field 'hosts' is required but was not set` to appear in the linter pr checks. Do you think the same fix is applicable here?

**Review by karianna — APPROVED** [review]
LGTM once linter fix is done

**Comment by steelhead31** [comment]
@gdams  I've fixed the linter/hosts sed issue on your branch :)

**Comment by steelhead31** [comment]
New VPC run : https://ci.adoptium.net/job/VagrantPlaybookCheck/1611/

**Comment by Haroon-Khel** [comment]
Still seeing `ERROR! the field 'hosts' is required but was not set`

I think since youve changed the vpc script itself, you'll need to kick off the job with this branch in the config of the job

Kicked off https://ci.adoptium.net/job/VagrantPlaybookCheck/1612/console

**Comment by Haroon-Khel** [comment]
It's not liking pulling from this branch either

```
10:44:05 hudson.plugins.git.GitException: Failed to fetch from https://github.com/gdams/openjdk-infrastructure.git
10:44:05 	at hudson.plugins.git.GitSCM.fetchFrom(GitSCM.java:1003)
10:44:05 	at hudson.plugins.git.GitSCM.retrieveChanges(GitSCM.java:1244)
10:44:05 	at hudson.plugins.git.GitSCM.checkout(GitSCM.java:1308)
```

**Review by steelhead31 — APPROVED** [review]
LGTM, I've checked/tested the nagios related changes, and they are all good.

**Pull request merged**